content/blog/esc-ga-announcement/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Announcing the General Availability of Pulumi ESC - Welcome to the new era of Secrets Management"
```

Comment:
I feel like this could probably be tightened even more - but one specific tactical suggestion:

```suggestion
title: "Announcing the General Availability of Pulumi ESC - Welcome to a New Era of Secrets Management"
```